lib/livebook/teams/app_deployment.ex

```diff
@@ -21,6 +23,8 @@ defmodule Livebook.Teams.AppDeployment do
     field :slug, :string
     field :sha, :string
     field :title, :string
+    field :multi_session, :boolean
+    field :access_type, Ecto.Enum, values: Livebook.Notebook.AppSettings.access_types()
```

Comment:
I am a bit worried about adding this information here, since we don't actually receive it from teams. What if we change Teams.Requests instead to receive both AppDeployment and AppSettings, so we don't have to add these new fields?

Reply:
Well, unless we want to start sending these fields back to Livebook.

@hugobarauna currently, after deploying to teams, we send the title/who/when deployed an application back to Livebook. This information is used when doing a new deploy on the same slug. If the slug is already in use, we show this information back to users.

Do you think it would be helpful to also show if the currently running app is password-protected or a multi-session? Or do we skip this information? I am thinking the title/who/when is enough but I am guessing it doesn't hurt to show more information?

Reply:
@josevalim, when I'm deploying a new version of an app, and I receive info from the currently deployed version, one way that info can be useful is to help me make sure I'm not making any unintentional config changes.

Let's imagine a scenario where the currently deployed app has a password, and when I'm deploying a new version, I disabled the password, but it was not intentional.

So yes, maybe showing more config info doesn't hurt. It may or may be not useful, depending on the user's context, but I don't think it would hurt.